kratos/processes/calculate_nodal_area_process.h

```diff
@@ -260,6 +224,28 @@ class CalculateNodalAreaProcess
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This method gets the current value of the NODAL_AREA
+     * @param rNode The node iterator to be get
+     * @return The current value of NODAL_AREA
+     */
+    double& GetAreaValue(NodeType& rNode);
+
+    /**
+     * @brief This method sets the current value of the NODAL_AREA to the given one
+     * @param rNode The node iterator to be get
+     * @param Value The current value of NODAL_AREA
+     */
+    void SetAreaValue(
+        NodeType& rNode,
+        const double Value
+        );
+
+    /**
+     * @brief This method sets the current value of the NODAL_AREA to the maximum
+     * @param itNode The node iterator to be set
+     */
+    void SetInitialValue(NodeIterator itNode);
```

Comment:
why do you sometimes pass an iterator and the other times a reference?

Reply:
Depending what is needed in the part of the code